cloudinit/config/cc_ubuntu_drivers.py

```diff
@@ -108,15 +100,12 @@ def install_drivers(cfg, pkg_install_func):
     # Register and set debconf selection linux/nvidia/latelink = true
     tdir = temp_utils.mkdtemp(needs_exe=True)
     debconf_file = os.path.join(tdir, "nvidia.template")
-    debconf_script = os.path.join(tdir, "nvidia-debconf.sh")
     try:
+        if not debconf.HAS_DEBCONF:
+            raise ValueError("python3-debconf must be installed.")
```

Comment:
We probably do not need this check here, we only need to ensure we add PRs which place `python3-debconf` in the ubuntu/bionic|focal|impish|devel branches `debian/control` files and `packages/debian/control.in` in this branch.

From an RPM perspective and debian-proper, they won't care to pull in this dependency as ubuntu_drivers module only runs on Ubuntu.

Reply:
Removed. Thanks for the clarification. I will make those PRs after this one is merged.